app/models/saved_claim/coe_claim.rb

```diff
@@ -117,7 +117,6 @@ def relevant_prior_loans(form_copy)
         'propertyCounty' => '',
         'propertyZip' => property_zip,
         'propertyZipSuffix' => property_zip_suffix || ''
-        # 'willRefinance' => loan_info['propertyAddress']['willRefinance'] || false
```

Comment:
@rjohnson2011 This line was previously commented out, but I'm removing it in this PR because it serves no purpose. I'm not sure it ever served a purpose 🤔.

Reply:
@rjohnson2011 Just checking, are you unwittingly commenting on the code prior to my changes?